src/Components/v2/Carousel.tsx

```diff
@@ -17,12 +19,32 @@ interface Props {
   renderLeftArrow?: Arrow
   renderRightArrow?: Arrow
   onArrowClick?: () => void
+  lastSlideVisible?: boolean
+  firstSlideVisible?: boolean
```

Comment:
This sort of logic can be inferred via `renderLeftArrow` and `renderRightArrow`.

Reply:
```      {props.renderLeftArrow
        ? props.renderLeftArrow({
            Arrow: LeftArrow,
            getSlick: () => slickRef,
            ...props,
          })
        : !props.firstSlideVisible && <LeftArrow />}```

Yeah I'm a bit confused about the `renderLeftArrow` as the arrow is being rendered if it's `false`.

Reply:
So something like (this is pseudocode, will have to play with it by looking at api): 

```jsx
<Carousel
  ...
  renderLeftArrow={slideProps => {
    if (slideProps.slideIndex === 0) { // or something, not sure the example api
       return null // don't render the arrow
    } else {
      return <slideProps.Arrow />
    }
  }}
/>
```